src/app/shared/main-navigation/mat-main-nav/menu-items.const.ts

```diff
@@ -93,17 +93,17 @@ export const menuItems: IMenuItem[] = JSON.parse(
                 "description": ""
             },
             {
-                "type": 0,
+                "type": 9,
                 "route": "/concerns",
                 "name": "Concerns",
                 "description": ""
             },
             {
-                "type": 0,
+                "type": 9,
                 "route": "/connections",
                 "name": "Connections",
                 "description": ""
-            }                    
+            }                   
```

Comment:
Can these be removed too?

Reply:
I tried to run the programme by removing these. Unfornately it did not work without these. That's why I was bound to change the value from 0 to 9
Not sure if there is other way.

Reply:
Sorry @Judge40 It wa my bad understanding. I was missing something yesterday. Thanks to @CaiWillis It is now resolved.
I have removed the concern and connection.